src/README.md

```diff
@@ -26,7 +26,7 @@ server.post('/', (req, res) => {
   smartApp.handleHttpCallback(req, res);
 });
 
-server.post('/slashCommand', async (req, res) => {
+server.post('/slack', async (req, res) => {
```

Comment:
This example code doesn't match up with the exposed endpoints of the included code